### ISSUE TITLE
Quick search missing after upgrade to 1.5?

### ISSUE THREAD

**Issue by rsommer** [cloudhut/kowl#283]
After upgrading to 1.5 (docker release), the quick search box and the "Hide internal topics" checkbox on the topic page are gone.

As of commit bdf42220fd8f83e0aa4bdaa8e4722daf2270dea0, the quick search box is gone.
As of commit 54f78522279f61678c508c22024be15731aa52ad, the "Hide internal topics" is gone.

Is this the intended behaviour or a bug?

**Comment by weeco** [comment]
The quick search is now part of the table header. We've made that change for two reasons, the first reason is that it was not clear what the quick search would search for (only topic name, cleanup.policy etc) and the other reason was a technical one

<img width="879" alt="Screenshot 2021-11-18 at 11 28 12" src="https://user-images.githubusercontent.com/23424570/142398532-459dad49-e687-45b9-9e4d-5c5ea09748a7.png">

I think the internal topics toggle has been removed as a result of this. I suspected that this change wouldn't be very intuitive for some users, we'll spend some time thinking about how to improve this.
